fineract-provider/dependencies.gradle

```
@@ -26,6 +26,14 @@ dependencies {
     providedRuntime("org.springframework.boot:spring-boot-starter-tomcat")
     
     spotbugsPlugins "jp.skypencil.findbugs.slf4j:bug-pattern:1.4.2@jar"
```

Since this the second plugin, it is preferable to change the implementation like this so that other comments can just be appended to the end.
That way the code looks better, if you get what I'm saying.

```suggestion
    spotbugsPlugins (
                   "jp.skypencil.findbugs.slf4j:bug-pattern:1.4.2@jar",
                   "com.h3xstream.findsecbugs:findsecbugs-plugin:1.8.0"
    )
```

---

This seems to have meanwhile been resolved.